message/job.py

```diff
@@ -25,23 +25,30 @@ class Message:
     A class that represents an AMQ Message.
     Messages can be serialized and deserialized for sending messages to and from AMQ
     """
+    # Note: Attributes default to None, but when sent to the DB, they cannot be null.
+    #   Perhaps could add check in QueueClient before sending, or change the defaults here.
```

Comment:
Really we want message to be nullable in the database. can you change this in the model? (`WebApp/autoreduce_webapp/reduction_viewer/models.py :: L65`)